src/SubscribeMe/Subscriber/BrevoSubscriber.php

```diff
@@ -195,4 +196,51 @@ public function sendTransactionalEmail(array $emails, string|int $emailTemplateI
             throw new CannotsendTransactionalEmailException($exception->getResponseBody()['message']);
         }
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function unsubscribe(string $email): bool
+    {
+        try {
+            if (!is_string($this->getApiKey())) {
+                throw new ApiCredentialsException();
+            }
+
+            if (!is_string($this->getContactListId())) {
+                throw new CannotSubscribeException('Contact list id is required for subscribe');
+            }
+
+            $body = [
+                'emails' => [$email],
+            ];
+
+            $bodyStreamed = $this->getStreamFactory()->createStream(json_encode($body, JSON_THROW_ON_ERROR));
+
+            $uri = 'https://api.brevo.com/v3/contacts/lists/listId/contacts/remove';
```

Comment:
```suggestion
            $uri = 'https://api.brevo.com/v3/contacts/lists/listId/contacts/remove';
```

Are you sure of `listId` hardocoded ??